content/en/docs/instrumentation/js/resources.md

```diff
@@ -0,0 +1,247 @@
+const {
+  dockerCGroupV1Detector,
+} = require("@opentelemetry/resource-detector-docker");
+
+// For troubleshooting, set the log level to DiagLogLevel.DEBUG
+diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.DEBUG);
+
+const sdk = new opentelemetry.NodeSDK({
+  traceExporter: new opentelemetry.tracing.ConsoleSpanExporter(),
+  instrumentations: [getNodeAutoInstrumentations()],
+  resourceDetectors: [dockerCGroupV1Detector],
+});
+
+sdk.start();
+```
+
+Build your docker image:
+
+```sh
+docker build . -t nodejs-otel-getting-started
+```
+
+Run your docker container:
+
+```sh
+$ docker run --rm -p 8080:8080 nodejs-otel-getting-started
+@opentelemetry/api: Registered a global for diag v1.2.0.
+...
+Listening for requests on http://localhost:8080
+DockerCGroupV1Detector found resource. Resource {
+  attributes: {
+    'container.id': 'fffbeaf682f32ef86916f306ff9a7f88cc58048ab78f7de464da3c320ldb5c54'
+  }
+}
+```
+
+The detector has extracted the `container.id` for you. However you might
+recognize that in this example the process attributes and the attributes set via
+an environment variable are missing: By setting the `resourceDetectors` list you
+overwrite the default value. To add them back update your `tracing.js` like the
+following:
+
+```javascript
+const opentelemetry = require("@opentelemetry/sdk-node");
+const {
+  getNodeAutoInstrumentations,
+} = require("@opentelemetry/auto-instrumentations-node");
+const { diag, DiagConsoleLogger, DiagLogLevel } = require("@opentelemetry/api");
+const {
+  dockerCGroupV1Detector,
+} = require("@opentelemetry/resource-detector-docker");
+const { envDetector, processDetector } = require("@opentelemetry/resources");
+
+// For troubleshooting, set the log level to DiagLogLevel.DEBUG
+diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.DEBUG);
+
+const sdk = new opentelemetry.NodeSDK({
+  traceExporter: new opentelemetry.tracing.ConsoleSpanExporter(),
+  instrumentations: [getNodeAutoInstrumentations()],
+  resourceDetectors: [envDetector, processDetector, dockerCGroupV1Detector],
+});
+
+sdk.start();
+```
+
+Rebuild your image and run your container once again:
+
+```shell
+docker run --rm -p 8080:8080 nodejs-otel-getting-started
+@opentelemetry/api: Registered a global for diag v1.2.0.
+...
+Listening for requests on http://localhost:8080
+EnvDetector found resource. Resource { attributes: {} }
+ProcessDetector found resource. Resource {
+  attributes: {
+    'process.pid': 1,
+    'process.executable.name': 'node',
+    'process.command': '/usr/src/app/app.js',
+    'process.command_line': '/usr/local/bin/node /usr/src/app/app.js',
+    'process.runtime.version': '18.9.0',
+    'process.runtime.name': 'nodejs',
+    'process.runtime.description': 'Node.js'
+  }
+}
+DockerCGroupV1Detector found resource. Resource {
+  attributes: {
+    'container.id': '654d0670317b9a2d3fc70cbe021c80ea15339c4711fb8e8b3aa674143148d84e'
+  }
+}
+...
+```
+
+## Visualising resources in Jaeger
```

Comment:
While helpful, I'm not sure if this is something we should document here. Even though it's OSS and "blessed" by otel, jaeger is just one backend. Other backends may treat resource attributes as just fields in the schema of whatever dataset traces are sent to, for example.

Reply:
I'd find it would be helpful to document the concept relations between OTEL and common backends, and how those backends are visualizing OTEL data. I've heard complaints about the confusion of misalignment of terms between OTEL and those backends. Though, this can be documented somewhere else.

Reply:
I removed that section.

> Though, this can be documented somewhere else.

Yes, we should have that in the Concept section or somewhere more general.